routes/api/contacts.js

```diff
@@ -1,25 +1,87 @@
-const express = require('express')
-
-const router = express.Router()
+const express = require('express');
+const router = express.Router();
+const Contacts = require('../../models/contacts.js');
```

Comment:
```suggestion
const contacts = require('../../models/contacts.js');
```
Od dużej litery zwykle oznaczamy class name w js.